pkg/cvo/cvo.go

```
@@ -813,6 +815,9 @@ func (b *resourceBuilder) builderFor(m *manifest.Manifest, state payload.State)
 func (b *resourceBuilder) Apply(ctx context.Context, m *manifest.Manifest, state payload.State) error {
 	builder, err := b.builderFor(m, state)
 	if err != nil {
+		if meta.IsNoMatchError(errors.Cause(err)) {
+			return nil
```

/hold

I'd rather fast-fail instead of ignoring this error.